pkg/get/get_test.go

```diff
@@ -1133,6 +1133,58 @@ func Test_DownloadK9s(t *testing.T) {
 	}
 }
 
+func Test_DownloadEtcd(t *testing.T) {
+	tools := MakeTools()
+	name := "etcd"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v3.4.27"
+
+	tests := []test{
+		{
+			os:      "linux",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     `https://github.com/etcd-io/etcd/releases/download/v3.4.27/etcd-v3.4.27-linux-amd64.tar.gz`,
+		},
+		{
+			os:      "darwin",
```

Comment:
could you add a test for darwin arm?

Reply:
there are no binaries available for darwin arm unfortunately (e.g. https://github.com/etcd-io/etcd/releases/tag/v3.4.27)

Reply:
is there a proper way to indicate in the template that a combination is not supported?

Reply:
If there's no binary for an OS/arch mix, then we usually won't add a test.

Reply:
cc @klutchell

Reply:
My mistake, recent releases like v3.5.9 do include darwin arm64 binaries so I added the test cases and re-ran all tests.